buildspec.yml

```diff
@@ -0,0 +1,32 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      nodejs: 20
```

Comment:
Minor: I suggest having a `.nvmrc` in the base directory with a node version matching the above.

Reply:
Done!